dev/com.ibm.ws.microprofile.metrics.common/resources/com/ibm/ws/microprofile/metrics/resources/Metrics.nlsprops

```diff
@@ -59,6 +59,11 @@ internal.error.CWMMC0006E=CWMMC0006E: An internal error occurred.
 internal.error.CWMMC0006E.explanation=The application server experienced an internal error.
 internal.error.CWMMC0006E.useraction=Gather a set of traces and open a new issue against your application server.
 
+#Information regarding libraryRef attribute taking effect only after server restart
+configurationChange.info.CWMMC0007I=CWMMC0007I: A configuration change was detected for the MicroProfile Metrics feature. If a change was made to the libraryRef attribute, a server restart is required for the change to take effect.
+configurationChange.info.CWMMC0007I.explanation=Due to the way that the MicroProfile Metrics feature loads the implementation binary files, a configuration change to the libraryRef attribute is not possible during run time. Reloading the library and its contents that are specified by the libraryRef attribute during run time erases all metric data. A change to the libraryRef attribute requires a server restart for the change to take effect.
```

Comment:
The reason we can't change the library on the fly is also that the metrics are registered at app startup.